src/styles/utils/_reset.scss

```diff
@@ -0,0 +1,14 @@
+body,
+main,
+section,
+div,
+ul,
+p,
+h2 {
+  margin: 0;
+  padding: 0;
+}
```

Comment:
All of this element don't have any default paddings
Also main, section and div don't have both default margins and paddings